dotcom-rendering/src/web/layouts/LiveLayout.tsx

```diff
@@ -175,6 +198,17 @@ const LiveGrid = ({ children }: { children: React.ReactNode }) => (
 	</div>
 );
 
+const doNotDisplayInWide = css`
+	${from.desktop} {
+		display: none;
+	}
+`;
+const doNotDisplayInNarrow = css`
+	${until.desktop} {
+		display: none;
+	}
+`;
```

Comment:
You can replace these with `Hide`. In fact, there's a [component in Source](https://github.com/guardian/source/blob/21f925531df0ddef06923efc9357d91de64a48cf/packages/%40guardian/src-layout/Hide/Hide.tsx) we should start to use for this.

Reply:
Wow that's great, I didn't know about the `Hide` component 😄  while looking I realised there's also a `Hide` component within dcr but I used the source one as you suggested

Reply:
The version of Hide in Source has a better api than DCR's.

DCR: `<Hide when="above" breakpoint="leftCol">`
Source: `<Hide above="leftCol">`

They won on that one!